pyproject.toml

```diff
@@ -127,13 +140,52 @@ lint.unfixable = [
 # Unlike Flake8, default to a complexity level of 10.
 lint.mccabe.max-complexity = 10
 
+[tool.flake8]
```

Comment:
Why add `flake8` and `pylint` when ruff is already there?

Reply:
You're right. This is just due to leftover flake8 and pylint integration in my neovim. I've updated it to use ruff, and only ruff, and removed flake8 and pylint from the pyproject.toml.